cvat/apps/engine/cache.py

```diff
@@ -216,8 +220,13 @@ def _create_and_set_cache_item(
     ) -> DataWithMime:
         timestamp = django_tz.now()
         item_data = create_callback()
-        item_data_bytes = item_data[0].getvalue()
-        item = (item_data[0], item_data[1], cls._get_checksum(item_data_bytes), timestamp)
+        data_to_be_cached, mime_type = item_data
+        if data_to_be_cached is None:
```

Comment:
`create_callback()` is not expected to return `None`. If it's not an error, it should return an empty `BytesIO` object.

Reply:
That makes sense, done